### ISSUE TITLE
.editorconfig file

### ISSUE THREAD

**Issue by igitur** [babun/babun#310]
I propose to add a `.editorconfig` file which will specify the type of line endings to use, whether white space should be trimmed at the end of lines, what kind of indentation to use, etc. See www.editorconfig.org for full spec.

I propose the following initial `.editorconfig` file. Do you agree? The line ending change for the .bat file affects all lines in `install.bat`.

```
# top-most EditorConfig file
root = true

# Unix-style newlines with a newline ending every file
# Indentation style of 4 spaces
[*]
end_of_line = lf
trim_trailing_whitespace = true
insert_final_newline = true
indent_style = space
indent_size = 4

# Windows-style newlines
[*.bat]
end_of_line = crlf
```

**Comment by tombujok** [comment]
I agree, it' s bigger topic since we also need http://git-scm.com/docs/gitattributes to set set it up on repo level too (e.g. line endings).
